stix_shifter_modules/azure_sentinel/stix_translation/json/to_stix_map.json

```diff
@@ -501,7 +497,7 @@
       "object": "alert"
     },
     "status": {
-      "key": "x-msazure-sentinel-alert.status",
+      "key": "x-microsoft-graph-alert.status",
```

Comment:
If you are changing the custom object to microsoft graph, shouldn't you change all instances of `x-msazure-sentinel-alert`?

Reply:
moved all instances of x-msazure-sentinel-alert under `x-ibm-finding`